invokeai/frontend/web/src/features/nodes/store/util/findConnectionToValidHandle.ts

```diff
@@ -12,7 +12,7 @@ import { getIsGraphAcyclic } from './getIsGraphAcyclic';
 const isValidConnection = (
   edges: Edge[],
   handleCurrentType: HandleType,
-  handleCurrentFieldType: FieldType,
+  handleCurrentFieldType: FieldType | string,
```

Comment:
nit: Here and in several other places in this PR we use `FieldType | string`, which is equivalent to just `string`.

I'd tend to just use the `string` type. Unless you think including `FieldType` improves readability?

Reply:
Yep, just for readability. TS also widens it to string, so when you use intellisense, you also see string. 

I could be convinced it's less readable than just string, though.